otcextensions/osclient/cbr/v3/policy.py

```diff
@@ -0,0 +1,534 @@
+            '--enable',
+            action='store_true',
+            help=_('Enables CBR Policy.')
+        )
+        parser.add_argument(
+            '--pattern',
+            metavar='<pattern>',
+            action='append',
+            dest='patterns',
+            help=_('Scheduling rule.\n'
+                   'Repeat Option for multiple rules.')
+        )
+        parser.add_argument(
+            '--day-backups',
+            metavar='<day_backups>',
+            type=int,
+            help=_('Specifies the number of retained daily backups.\n'
+                   'Ranges from 0 to 100.\n'
+                   'If this parameter is configured, timezone is '
+                   'mandatory.')
+        )
+        parser.add_argument(
+            '--week-backups',
+            metavar='<week_backups>',
+            type=int,
+            help=_('Specifies the number of retained weekly backups.\n'
+                   'Ranges from 0 to 100.\n'
+                   'If this parameter is configured, timezone is '
+                   'mandatory.')
+        )
+        parser.add_argument(
+            '--month-backups',
+            metavar='<month_backups>',
+            type=int,
+            help=_('Specifies the number of retained monthly backups.\n'
+                   'Ranges from 0 to 100.\n'
+                   'If this parameter is configured, timezone is '
+                   'mandatory.')
+        )
+        parser.add_argument(
+            '--year-backups',
+            metavar='<year_backups>',
+            type=int,
+            help=_('Specifies the number of retained yearly backups.\n'
+                   'Ranges from 0 to 100.\n'
+                   'If this parameter is configured, timezone is '
+                   'mandatory.')
+        )
+        parser.add_argument(
+            '--timezone',
+            metavar='<timezone>',
+            help=_('Timezone where the user is located, e.g. UTC+08:00.')
+        )
+        parser.add_argument(
+            '--max-backups',
+            metavar='<max_backups>',
+            type=int,
+            help=_('Maximum number of retained backups.\n'
+                   'Can be -1, backups will not cleared or ranges\n'
+                   'from 0 to 99999.')
+        )
+        parser.add_argument(
+            '--retention-duration-days',
+            metavar='<retention_duration_days>',
+            type=int,
+            help=_('Duration of retaining a backup, in days.\n'
+                   '-1 indicates that the backups will not be '
+                   'cleared based on the retention duration.\n'
+                   'Ranges from 1 to 99999.')
+        )
+        return parser
+
+    def take_action(self, parsed_args):
+
+        attrs = {}
+
+        if parsed_args.name:
+            attrs['name'] = parsed_args.name
+        if parsed_args.enable:
+            attrs['enabled'] = True
+        if parsed_args.disable:
+            attrs['enabled'] = False
```

Comment:
attrs['disabled'] maybe

Reply:
No, I can't do it. I want to modify the 'enabled' param of the policy: https://docs.otc.t-systems.com/en-us/api/cbr/UpdatePolicy.html